tools/hmaptool/BUILD

```diff
@@ -0,0 +1,61 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+load("//apple:macos.bzl", "macos_command_line_application")
+
+package(default_visibility = ["//apple/internal:__subpackages__"])
+
+HMAP_COPTS = [
+    "-DHASH_FUNCTION=HASH_MUR",
+    "-DHASH_USING_NO_STRICT_ALIASING",
+    "-fno-strict-aliasing",
+]
+
+cc_library(
+    name = "lines",
+    srcs = ["lines.c"],
+    hdrs = ["lines.h"],
+    copts = ["-Wno-parentheses"],
+)
+
+cc_library(
+    name = "hmap",
+    srcs = ["hmap.c"],
+    hdrs = ["hmap.h"],
+    copts = HMAP_COPTS,
+)
+
+cc_binary(
+    name = "hmaptool_c",
+    srcs = [
+        "hmaptool.c",
+        "uthash.h",
+    ],
+    copts = HMAP_COPTS,
+    # Used by the rule implementations, so it needs to be public; but
+    # should be considered an implementation detail of the rules and
+    # not used by other things.
+    visibility = ["//visibility:public"],
+    deps = [
+        ":hmap",
+        ":lines",
+    ],
+)
+
+swift_library(
+    name = "BinaryHeaderMapTool",
+    srcs = [
+        "BinaryHeaderMap.swift",
+        "BinaryHeaderMapEncoder.swift",
+        "BinaryHeaderMapTool.swift",
+    ],
+)
+
+macos_command_line_application(
```

Comment:
Maybe switch to `swift_binary` wrapped by an `apple_universal_binary` instead? I think the minimum version setting should be set by downstream users.

Reply:
Makes sense, I'll update!